docs/pages/connect-your-client/vnet.mdx

```diff
@@ -33,10 +41,12 @@ have `tcp://` as the protocol in their addresses.
 Click "Connect" next to the TCP app. This starts VNet if it's not already running. Alternatively,
 you can start VNet through the connection list in the top left.
 
+<Details title="First launch on macOS">
 During the first launch, macOS will prompt you to enable a background item for tsh.app. VNet needs
 this background item in order to configure DNS on your device. To enable the background item, either
 interact with the system notification or go to System Settings > General > Login Items and look for
 tsh.app under "Allow in the Background".
+</Details>
 
 ![VNet starting up](../../img/use-teleport/vnet-starting@2x.png)
```

Comment:
I feel like this image could be pulled into the "First launch on macOS" section if possible as it includes multiple elements that are macOS-only (the system notification and the in-app notification).

Reply:
done